game/scripts/npc/heroes/medusa.txt

```diff
@@ -0,0 +1,14 @@
+"DOTAHeroes"
+{
+  //=================================================================================================================
+  // HERO: Medusa
+  //=================================================================================================================
+  "npc_dota_hero_medusa"
+  {
+    "Ability14"		"special_bonus_unique_medusa_5"
+    "Ability15"		"special_bonus_mp_800" // replaces special_bonus_unique_medusa
```

Comment:
i'd make this like 700 since that is what it was before when it was a level 20 talent